cowprotocol/cowswap
# Eth flow/updated contract to 0.0.0 rc4

# Summary

Update ethflow contract to the split version of barn/prod

Placed a swap:
- tx https://goerli.etherscan.io/tx/0x7a8c475c84301f9ee977c70984d2136c500453c8ec4f1b707484b6949aaa8d0e
- order https://protocol-explorer.dev.gnosisdev.com/goerli/orders/0x1e76baa3eef0727665ab17cc8149b3c5312c26ec4ef1365340b3e3b032a593dcf55e40061fe3dffb7ae166b46a008dfc2ada510fffffffff

  # To Test

1. Place an order from this PR
* Should use the contract `0xF55E40061fe3dFfB7aE166B46A008dfc2aDA510f`
